packaging/deb/grafana-agent.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Monitoring system and forwarder
+Documentation=https://github.com/grafana/agent/blob/master/docs/README.md
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+Restart=always
+User=grafana-agent
+EnvironmentFile=/etc/default/grafana-agent
+ExecStart=/usr/bin/grafana-agent --config.file $CONFIG_FILE --log.level $LOG_LEVEL $CUSTOM_ARGS
+ExecReload=/bin/kill -HUP $MAINPID
+TimeoutStopSec=20s
```

Comment:
Is this the maximum amount of time before force killing the service? Can we add a comment here saying something along these lines?

> If running the Agent in scraping service mode, you may wish to override this value with something significantly more generous to allow the Agent to gracefully leave the cluster; 4800s is recommended.

(Ditto for the rpm service file)

Reply:
Good catch. It actually covers multiple timeout cases but the one you mentioned is included. I think that comment would be valuable. Is the `20s` default okay for the basic configuration installed as part of the package?

Reply:
20s should be fine for the basic configuration, if the Agent stops too early it'll still have data in the WAL that it can send on the next startup.